flink-contrib/docker-flink/flink/Dockerfile

```diff
@@ -25,8 +25,8 @@ RUN cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys && chmod 600 ~/.ssh/*
 ##Flink 0.10.0 Installation
 ###Download:
 RUN mkdir ~/downloads && cd ~/downloads && \
-    wget -q -O - http://apache.spd.co.il/flink/flink-0.10.0/flink-0.10.0-bin-hadoop27-scala_2.11.tgz | tar -zxvf - -C /usr/local/
-RUN cd /usr/local && ln -s ./flink-0.10.0 flink
+    wget -q -O - http://mirror.switch.ch/mirror/apache/dist/flink/flink-0.10.1/flink-0.10.1-bin-hadoop27-scala_2.11.tgz| tar -zxvf - -C /usr/local/
```

Comment:
I think it's fine to switch the mirror. There was no particular reason for the mirror before. It would be neat to go through the mirrors list at http://www.apache.org/dyn/closer.lua/flink/flink-0.10.1/flink-0.10.1-bin-hadoop27-scala_2.11.tgz but not necessary to change that now.